colosseum/constants.py

```diff
@@ -54,19 +33,13 @@ def validate(self, value):
 
     def __str__(self):
         choices = [str(c).lower().replace('_', '-') for c in self.constants]
-        if self.length:
-            choices.append("<length>")
-        if self.percentage:
-            choices.append("<percentage>")
-        if self.integer:
-            choices.append("<integer>")
-        if self.number:
-            choices.append("<number>")
-        if self.color:
-            choices.append("<color>")
+        for validator in self.validators:
+            choices.append(validator.name)
```

Comment:
This is the only thing that would go away with the function/closures approach.

Thoughts @freakboy3742 ?